src/pkey/clu_pkey.c

```diff
@@ -486,9 +486,10 @@ int wolfCLU_pKeySetup(int argc, char** argv)
                     }
                 }
                 else {
+                    derSz = wolfSSL_i2d_PublicKey(pkey, &der);
```

Comment:
Should this be private key here instead of public key? Should use wolfCLU_pKeytoPriKey (works around an ECC internal not set with PKEY struct)

Reply:
Removed as it was fixed in https://github.com/wolfSSL/wolfCLU/pull/60